compose/bin/log

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+LOGS_PATH="/var/www/html/var/log"
+CONTAINER_NAME="$(basename "$(pwd)")-phpfpm-1"
+
+echo -e "\033[94mDeveloped by \033[34mYevhen \033[93mZvieriev\033[0m"
+
+function view_log() {
+    local log_file="$1"
+    docker exec -it "$CONTAINER_NAME" cat "$LOGS_PATH/$log_file"
+}
+
+function view_log() {
```

Comment:
it's just duplicate of the same function as above

Reply:
I fixed this, thx